src/modules/dataman/dataman.c

```diff
@@ -949,7 +949,15 @@ _file_initialize(unsigned max_offset)
 		close(dm_operations_data.file.fd);
 
 		if (incompat) {
+#ifdef CONFIG_ARCH_BOARD_AEROCORE2
```

Comment:
@smithandrewc - 

Great! I think that solution will work for this configuration. It will not on other platforms that support the hardfault_log.  

I have tested this with QGC.  (You will need the PID added to the inf file for windows,)

I did the Sensor calibration but I did not test the radio calibrations.

The Tone alarm is signaling error due to the lack of SDCARD and .
See:
```
# Run no SD alarm
if [ $LOG_FILE == /dev/null ]
```
in rcS.

Also, I noticed the LEDs are off. Did you want to add the LED lib and support the all 3 LEDs on the HW? If so, you will need the new led lib and an unfortunate #ifdef in commander.cpp control_status_leds(). See the ```cmake/configs/nuttx_px4fmu-v4_default.cmake``` for the led parts and ```src/drivers/boards/px4fmu-v4/px4fmu_led.c``` for a nice LED driver)

Thank you for getting this done.